Makefile.in

```diff
@@ -93,7 +93,14 @@ DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/Makefile.am \
 	$(top_srcdir)/libltdl/config/ltmain.sh \
 	$(top_srcdir)/libltdl/config/missing
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
-am__aclocal_m4_deps = $(top_srcdir)/configure.ac
+am__aclocal_m4_deps = $(top_srcdir)/libltdl/m4/argz.m4 \
```

Comment:
I don't. How do you do that? I just did make, then git push. Shouldn't these files be in the .gitignore file?